calc/async_decoupler.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
```

Comment:
Each project seems to have its own convention here; I think our other projects do it this way.